pyproject.toml

```diff
@@ -71,28 +50,27 @@ charset-normalizer = ">=2.1.0,<3.1"
 # Use the Python 3.11 compatible API: https://github.com/python/importlib_metadata#compatibility
 importlib_metadata = { version = ">=4.13,<5"}
 
-[tool.poetry.dev-dependencies]
-ipython = "^7.2"
+[tool.poetry.group.dev.dependencies]
```

Comment:
If we're using this feature, should we set the minimum poetry version?

Reply:
I didn't know it was possible, I'll give it a try

Reply:
I found this: https://github.com/python-poetry/poetry/issues/3316

Reply:
ah. I thought we could do that through build section. Then maybe we can add a description in the `README.md`?